hexa/workspaces/templates/workspaces/mails/invite_external_user.html

```diff
@@ -0,0 +1,12 @@
+{% load i18n %}
+{% autoescape off %}
+{% blocktranslate %}You have been invited to join the workspace <b>{{workspace}}</b> by {{owner}}.{% endblocktranslate %}
+
+{% translate "Use the following link to proceed:" %}
+
+<a href="{{ workspace_signup_url }}">{% translate "Accept invitation" %}</a>
```

Comment:
Since we do not handle yet the case where you accept the invitation, I think it would be better to write "Create your account" or something like this

Reply:
Yes, you're right